device_doctor/lib/src/utils.dart

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:process/process.dart';
+
+/// Virtual current working directory, which affect functions, such as [exec].
+String cwd = Directory.current.path;
+
+ProcessManager processManager = LocalProcessManager();
```

Comment:
Same comment as elsewhere: globals are going to make this code difficult to refactor later

Reply:
Changed to use local variables instead.